pkg/auth/mtls_authhandler.go

```diff
@@ -0,0 +1,279 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package auth
```

Comment:
Do you plan to add tests for this file?

Reply:
There is this file pkg/auth/mtls_authhandler_test.go ?

Reply:
Oh sorry I missed it, probably because GH UI hit it under "large diff" :facepalm: